java/src/main/java/org/digma/intellij/plugin/idea/psi/java/AbsJaxrsFramework.java

```diff
@@ -114,59 +112,102 @@ private boolean isJaxRsHttpRelevant() {
         return jaxrsPathAnnotationClass != null;
```

Comment:
is AbsJaxrsFramework shortcut for AbstractJaxrsFramework ? isn't that confusing with abs functions? IMP shortcut for Abstract is not clear.

Reply:
i'll rename